### PR TITLE
Add the possibility to order layers/visualisations

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/app/modules/map-config/components/visualisations/visualisations.component.html [hunk 1]
```diff
@@ -7,31 +7,34 @@
...
                 <span style="display: inline-block; margin-right: 5px;" *ngFor="let l of element.layers"> • {{l}}</span>
             </div>
         </td>
     </ng-container>
 
     <ng-container matColumnDef="displayed">
-        <th style="width: 8%; text-align: center;" mat-header-cell *matHeaderCellDef>{{'Displayed by default' | translate}}</th>
-        <td style="text-align: center;" mat-cell *matCellDef="let element"> 
+        <th class="th-display" mat-header-cell *matHeaderCellDef>{{'Displayed by default' | translate}}</th>
+        <td class="th-display" mat-cell *matCellDef="let element">
```
Comment (sebbousquet) [inline src/app/modules/map-config/components/visualisations/visualisations.component.html:31]: `class="th-display"` => on a `td` element is it normal ?

File: src/app/modules/map-config/components/visualisations/visualisations.component.html [hunk 2]
```diff
@@ -7,31 +7,34 @@
...
+        <th class="th-display" mat-header-cell *matHeaderCellDef>{{'Displayed by default' | translate}}</th>
+        <td class="th-display" mat-cell *matCellDef="let element"> 
             <mat-checkbox [checked]="element.displayed" [disabled]="true"></mat-checkbox>
         </td>
     </ng-container>
     <ng-container matColumnDef="action">
-        <th mat-header-cell *matHeaderCellDef>{{'Actions' | translate}}</th>
-        <td mat-cell *matCellDef="let element">
+        <th class="th-action" mat-header-cell *matHeaderCellDef>{{'Actions' | translate}}</th>
+        <td class="th-action" mat-cell *matCellDef="let element">
```
Comment (sebbousquet) [inline src/app/modules/map-config/components/visualisations/visualisations.component.html:37]: `class="th-action"` => on a `td` element is it normal ?

File: src/app/modules/map-config/components/visualisations/visualisations.component.html (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/modules/map-config/components/visualisations/visualisations.component.html]: fixed

File: src/app/modules/map-config/components/visualisations/visualisations.component.html (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/modules/map-config/components/visualisations/visualisations.component.html]: fixed

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**